samples/csharp/MicrosoftGraphSDK/Module.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Graph.Beta;
+using Microsoft.Graph.Beta.Models;
+
+namespace MicrosoftGraphSDK
+{
+    /// <summary>
+    /// Graph SDK endpoints for modules
+    /// </summary>
+    public class Module
+    {
+        public static async Task<EducationModule> CreateAsync(
+            GraphServiceClient client,
+            string classId,
+            string displayName,
+            string description)
+        {
+            try
+            {
+                var requestBody = new EducationModule
+                {
+                    DisplayName = displayName,
+                    Description = description,
+                };
+
+                return await client.Education
+                    .Classes[classId]
+                    .Modules.PostAsync(requestBody);
+            }
+            catch (Exception ex) {
+                throw new GraphException($"CreateAsync call: {ex.Message}", ex, classId);
+            }
+        }
+
+        public static async Task<EducationModuleResource> PostResourceAsync(
+            GraphServiceClient client,
+            string classId,
+            string moduleId,
+            EducationModuleResource resource)
+        {
+            try
+            {
+                return await client.Education
+                    .Classes[classId]
+                    .Modules[moduleId]
+                    .Resources.PostAsync(resource);
+            }
+            catch (Exception ex)
+            {
+                throw new GraphException($"PostResourceAsync call: {ex.Message}", ex, classId);
+            }
+        }
+
+        public static async Task<EducationModule> PublishAsync(
+            GraphServiceClient client,
+            string classId,
+            string moduleId)
+        {
+            try
+            {
+                return await client.Education
+                    .Classes[classId]
+                    .Modules[moduleId]
+                    .Publish
+                    .PostAsync();
+            }
+            catch (Exception ex)
+            {
+                throw new GraphException($"PublishAsync call: {ex.Message}", ex, classId);
```

Comment:
log all the parameters here, you are only logging the classId

Reply:
logged all the parameters.